tasks/src/test/java/org/example/aSerejaAndDima/ASerejaAndDima2Test.java

```diff
@@ -0,0 +1,94 @@
+package org.example.aSerejaAndDima;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class ASerejaAndDima2Test {
+    private final ASerejaAndDima2.ASerejaAndDima2Verifier aSerejaAndDima2Verifier = new ASerejaAndDima2.ASerejaAndDima2Verifier();
+    private final ASerejaAndDima2 aSerejaAndDima2 = new ASerejaAndDima2();
+    private static List<Integer> listOfCardsInsert;
+
+    @BeforeAll
+    static void insertListOfCards() {
+        listOfCardsInsert = List.of(1, 2, 3, 4, 5, 6, 7);
```

Comment:
zasadniczo w tak prozaicznym przypadku nie ma sensu robic tego w beforeAll. Bo dokladnie na to samo wyjdzie jak zainicjalizujesz to od razu w 12 linijce. I wtedy lista nie musi byc statyczna

Reply:
Chodziło mi tylko o czystą praktykę wykorzystania @BeforeAll